examples/08 Introduction to PVMT.ipynb

```diff
@@ -0,0 +1,330 @@
+      "text/plain": [
+       "[<pvmt.Group \"Power\"(1e241678-282c-46f3-abbd-e0cf34e743ee)>,\n",
+       " <pvmt.Group \"Power Level\"(a0cb5a23-955e-43d6-a633-2c4e66991364)>]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "[<pvmt.FloatPropertyValue \"Min\"(4c5eba40-fcc3-480f-b88f-e375dadc00e5)>,\n",
+       " <pvmt.FloatPropertyValue \"Max\"(196467a2-ecee-4f35-8987-6a898cd34616)>]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "[<pvmt.EnumerationPropertyValue \"Skill\"(7367c731-aabb-4807-9d74-7ef2c6482ec9)>]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "darkmagic_domain = model.pvext.domains[0]\n",
+    "display(darkmagic_domain)\n",
+    "display(darkmagic_domain.groups)\n",
+    "for grp in darkmagic_domain.groups:\n",
+    "    display(grp.properties)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The domain scope (i.e. what ArchitectureLayer) this extension can be apploed to is not yet processed further. Scopes have rules underneath to control on which ModelObject's this can be applied to.\n",
+    "This is not yet supported."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'[PROPERTY]DarkMagic.Power.Max<10000.0[/PROPERTY]\\n[ARCHITECTURE]LOGICAL[/ARCHITECTURE]'"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "level = darkmagic_domain.groups[-1]\n",
+    "level.scope"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Access PVMT property values\n",
+    "\n",
+    "The ground rule is that you can access PVMT via the `.pvmt` attribute on literally everything you can find in a model. Then the pattern is `domain.group.subgroup.[moresubgroups...].property` from which the pvmt can be indexed with:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's check Dumbledore's dark magic power level:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'Mighty'"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dumbledore = model.la.all_components.by_name(\"Prof. A. P. W. B. Dumbledore\")\n",
+    "dumbledore.pvmt[\"DarkMagic.Power Level.Skill\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In our test model we have more famous wizzards. Let us define a function to collect all property values and load them into a pandas.DataFrame:"
```

Comment:
```suggestion
    "In our test model we have more famous wizards. Let us define a function to collect all property values and load them into a pandas.DataFrame:"
```

From [Wiktionary](https://en.wiktionary.org/wiki/wizzard):

> **wizzard** (*plural* [wizzards](https://en.wiktionary.org/wiki/wizzards#English))
> 1. *Obsolete spelling of [wizard](https://en.wiktionary.org/wiki/wizard#English)*

We have enough obsolete things in our code already, let's try to not introduce more. ;)